target/consensource-api
# Add prometheus metrics endpoint for http request counting

## Proposed change/fix

@pa3ng changes to add Prometheus metrics endpoint and count the number of http requests made.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test
Spin the project up from the compose repo and ensure transactions can be processed.
Make a get request to `/prom_metrics` to ensure it returns data.
